docs/src/domains/steps/custom-non-production-domains.md

```diff
@@ -90,68 +88,68 @@ Downgrading your plan doesn't affect custom domains set on your production envir
 
 To add a custom domain to a non-production environment, follow these steps:
 
-1. Get the target for your non-production environment.
-
 {{< codetabs >}}
 +++
 title=Using the CLI
 +++
 
-Run the following command:
+1. To get the target for your non-production environment,
+   run the following command:
 
-```bash
-platform environment:info edge_hostname --environment {{< variable "ENVIRONMENT_NAME" >}}
-```
+   ```bash
+   platform environment:info edge_hostname --environment {{< variable "ENVIRONMENT_NAME" >}}
+   ```
+
+2. [Configure your DNS provider](../steps/_index.md#3-configure-your-dns-provider).
+   In particular, make sure your DNS record points to the target of your non-production environment.
+
+   {{< note >}}
+
+   Using the target of your production environment to configure your DNS provider is technically possible,
+   but Platform.sh recommends using the target of your non-production environment as a best practice.
+
+   {{< /note >}}
+
+3. Run a command similar to the following:
+
+   ```bash
+   platform domain:add staging.example.com --environment {{< variable "STAGING_ENVIRONMENT_ID" >}} --attach {{< variable "PRODUCTION_CUSTOM_DOMAIN_TO_ATTACH" >}}
+   ```
 
 <--->
 +++
 title=In the Console
 +++
 
-1. In the Console, open your non-production environment.
-2. Click **URLs** and copy the URL to your site excluding `https://`.
+1. Get the target for your non-production environment. To do so, open the environment in the Console
```

Comment:
We can now change this instruction with something easier for the user.
They can go to the new `Domains` tab of the environment settings.
They will find here a `Configure your domain` block, where there is the `CNAME target` to copy.

We should also change it on [this page](https://non-prod-custom-xmpenxa-652soceglkw4u.eu-3.platformsh.site/domains/steps.html#2-get-the-target-for-your-project).

Reply:
Done